content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -406,6 +406,9 @@ different Kubernetes components.
 | `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
 | `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
 | `WatchBookmark` | `true` | GA | 1.17 | - |
+| `WindowsEndpointSliceProxying` | `false` | Alpha | 1.19 | 1.20 |
+| `WindowsEndpointSliceProxying` | `true` | Beta | 1.21 | 1.21 |
+| `WindowsEndpointSliceProxying` | `true` | GA | 1.22| - |
```

Comment:
what about `EndpointSliceProxying`

also, it seems that the code is still in beta
```
pkg/features/kube_features.go:  EndpointSliceProxying featuregate.Feature = "EndpointSliceProxying"
pkg/features/kube_features.go:  WindowsEndpointSliceProxying featuregate.Feature = "WindowsEndpointSliceProxying"
pkg/features/kube_features.go:  EndpointSliceProxying:                          {Default: true, PreRelease: featuregate.Beta},
pkg/features/kube_features.go:  WindowsEndpointSliceProxying:                   {Default: true, PreRelease: featuregate.Beta},
```

Reply:
one thing to note, if it is GA, it needs to be removed from the first table as well

Reply:
This draft is still very much a WIP. I just opened the PR with a small change for tracking purposes.

Reply:
Gates have graduated in the code. And the gates have been removed from the first table.